automated/linux/usb/test-usb-2.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+source "../test_format.sh"
+
+dd if=/dev/urandom of=/tmp/testfile bs=512 count=1k
+md5sum /tmp/testfile > md5 
+cp /tmp/testfile /mnt && cp md5 /mnt
+cd /mnt && md5sum -c md5
```

Comment:
Who mounts the device and in which test or test precondition mounts it?

Reply:
Thanks, You are right. 
I fixed it. Now it is mounted after some checks...
I think it is better now, but it only tests sda1...